docs/samples/product_recommendation_demo_advanced.ipynb

```diff
@@ -135,6 +135,13 @@
             "! pip install feathr azure-cli  pandavro scikit-learn\n"
          ]
       },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "When running this notebook in synapse, you may get some errors or blocks installing above packages in one cell. Suggest to try installing them in seperate cells if meet some issues. Eg. ! pip install feathr, ! pip install  azure-cli , ! pip install  pandavro, ! pip install scikit-learn"
```

Comment:
This looks like a generic tips for synapse notebook rather than specific for this sample notebook right?
If so, add a FAQ or known issue section in docs would also be helpful.

Reply:
Will add a FAQ soon.